extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaSecurityIdentity.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.amazon.lambda.http;
+
+import java.security.Permission;
+import java.security.Principal;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import io.quarkus.security.credential.Credential;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.smallrye.mutiny.Uni;
+
+public class LambdaSecurityIdentity implements SecurityIdentity {
```

Comment:
Why do you need this class? Can't you just build a QuarkusSecurityIdentity with these attributes??